lab/src/Microsoft.SyndicationFeed/src/Rss20/Rss20ElementNames.cs

```diff
@@ -28,5 +28,13 @@ public static class Rss20ElementNames
         public const string Title = "title";
         public const string Url = "url";
         public const string Version = "version";
+        
```

Comment:
The separation is only to show the new elements. We can always merge them with the other ones.